metron-platform/metron-parsers/src/test/java/org/apache/metron/parsers/bro/BasicBroParserTest.java

```diff
@@ -1133,6 +1133,233 @@ public void testKnownDevicesBroMessage() throws ParseException {
         }
 
 	/**
```

Comment:
This extra /** is why the tests are failing

Reply:
Fixed with my latest commit - thanks.